.github/workflows/create_containers.yml

```
@@ -11,31 +11,25 @@ jobs:
-        include:
-          - distro: debian
-            runner: ubuntu-latest
-            cross: i386
-          - distro: debian
-            runner: ubuntu-24.04-arm
-            cross: null
-          - distro: debian
-            runner: ubuntu-24.04-arm
-            cross: s390x
+        # NOTE: values are joined by '-' for name of container image
+        - { DISTRO: arch, ARCH: amd64 }
```

actionlint (1.7.12)

[error] "matrix" section is sequence node but mapping node is expected

(syntax-check)